libs/blocks/gnav/gnav-appLauncher.js

```diff
@@ -0,0 +1,50 @@
+import { createTag, makeRelative } from '../../utils/utils.js';
+
```

Comment:
Another general thing - we should have test coverage for this file (unless you didn't write any because it's a temporary solution? :D )

Reply:
I will get to tests..